torch/nn/_intrinsic/qat/modules/conv_relu.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+from torch.nn.qat import Conv2d as QATConv2d
+from torch.nn._intrinsic import ConvReLU2d as NNConvReLU2d
+from torch.quantization.QConfig import default_qat_qconfig
+import torch.nn.functional as F
+
+class ConvReLU2d(QATConv2d):
+    r"""
+    A ConvReLU2d module is a fused module of Conv2d and ReLU, attached with
+    FakeQuantize modules for both output activation and weight for
+    quantization aware training.
+
+    We adopt the same interface as `torch.nn.Conv2d`, please see
+    https://pytorch.org/docs/stable/nn.html?highlight=conv2d#torch.nn.Conv2d
+    for documentation.
+
+    Similar to :class:`~torch.nn.Conv2d`, with FakeQuantize modules initialized to
+    default.
+
+    Attributes:
+        observer: fake quant module for output activation, it's called observer
+            to align with post training flow
+        weight_fake_quant: fake quant module for weight
+
+    """
+    __FLOAT_MODULE__ = NNConvReLU2d
+
+    def __init__(self, in_channels, out_channels, kernel_size, stride=1,
+                 padding=0, dilation=1, groups=1,
+                 bias=True, padding_mode='zeros',
+                 activation_fake_quant=default_qat_qconfig.activation,
+                 weight_fake_quant=default_qat_qconfig.weight):
+        super(ConvReLU2d, self).__init__(in_channels, out_channels, kernel_size,
+                                         stride=stride, padding=padding, dilation=dilation,
+                                         groups=groups, bias=bias, padding_mode=padding_mode)
+        self.observer = activation_fake_quant()
```

Comment:
Lets settle on a name: Observer is not correct. How about calling it 'activation_post_process': This can be either observers or fake quant.

Reply:
OK, we can change it later after everything settles. since this requires changing all references to  ".observer"